pkg/parse/event_handler.go

```diff
@@ -59,7 +59,6 @@ func (s *EventHandler) Handle(event events.Event) events.Result {
 	// This delays status update and sync events.
 	runFn := func(ctx context.Context, trigger string) ReconcileResult {
 		result := s.Reconciler.Reconcile(ctx, trigger)
```

Comment:
you can just return this directly, or remove the whole runFn wrapper

Reply:
comment is out of date, I realized we need the runFn to reset the sync timer whenever a reconcile is triggered by one of the other event types